web/themes/custom/novel/templates/layout/footer.html.twig

```diff
@@ -1,35 +1,35 @@
 <footer class="footer">
   <h2 class="hide-visually">Globale links</h2>
-  <div class="footer--mobile">
-    <div class="pagefold-parent-small internal-pagefold-parent">
-      <div class="pagefold-triangle-small pagefold-inherit-parent"></div>
+  <div class="footer__mobile">
+    <div class="pagefold-parent--small internal-pagefold-parent">
+      <div class="pagefold-triangle--small pagefold-inherit-parent"></div>
       <div>
         <h3 class="text-header-h4 mb-16">Åbningstider</h3>
         <p class="text-body-medium-regular mb-24">
           Bibliotekerne lorem ipsum consectetur, adipisci velit, sed quia non
           numquam eius modi tempora incidunt ut labore.
         </p>
-        <div class="footer-column--link mb-48">
+        <div class="footer__column-link mb-48">
           <a href="/" class="link-tag">Se bibliotekernes åbningstider</a>
         </div>
       </div>
       <ul class="accordion">
-        <li class="accordion-row">
-          <h3 class="accordion-header text-header-h4">
+        <li class="accordion__row">
+          <h3 class="accordion__header text-header-h4">
             <button
-              class="accordion-button px-48"
+              class="accordion__button px-48"
               aria-expanded="false"
               data-accordion-trigger="true"
             >
               Om Bibliotekerne
               <img
-                class="accordion-button--arrow"
-                src="/themes/custom/novel/assets/dpl-design-system/icons/basic/icon-triangle.svg"
+                class="accordion__button-arrow"
+                src="icons/basic/icon-triangle.svg"
```

Comment:
Please fix the url to the icon. The problem with the 404 resides here.

Reply:
👍